stdpipe/psf.py

```diff
@@ -352,8 +352,8 @@ def place_psf_stamp(image, psf, x0, y0, flux=1, gain=None):
 
     # Corresponding image pixels
     y1,x1 = np.mgrid[0:stamp.shape[0], 0:stamp.shape[1]]
-    x1 += np.int(np.round(x0) - np.floor(stamp.shape[1]/2))
-    y1 += np.int(np.round(y0) - np.floor(stamp.shape[0]/2))
+    x1 += utils.float2int_array((np.round(x0) - np.floor(stamp.shape[1]/2)))
```

Comment:
x0,y0 here are scalars, so probably just `int(np.round(x0) - np.floor(stamp.shape[1]/2))` would go?..

Reply:
Yes, if x0 and y0 are scalars it will work with just int()

Reply:
Could you please commit this variant (with just int, and without extra util function) - as I probably cannot commit to your branch. After that I will merge the PR

Reply:
I'll do it asap